### PR TITLE
Updates to Asset Management, Endpoint, Assessments, and Public Scanning

### PR DESCRIPTION
**Pull request by JTAGulous**
Updated Asset Managmenet BP filename and title, alongside making some minor changes.

Updated endpoint protection BP with terms and just general cleanup. Added some CIS Controls references.

Added some CIS Controls and tools to public_asset_scanning.rst

Added some CIS Controls and tools to formal_assessment.rst

### CODE REVIEW

File: docs/source/BPs/endpoint_protection.rst [hunk 1]
```diff
@@ -9,28 +9,30 @@
...
+:term:`Endpoint Protection` is security software that is deployed on workstations and servers, which are commonly referred to as “endpoints.” A common name for this is :term:`Endpoint Detection and Response`, or EDR. EDR collects technical data from these endpoints and transmits it back to the vendor or a local server. The data is then analyzed for suspicious patterns and threats.
 
 If a threat is identified, it is blocked and an alert is generated. Administrators can typically view alerts through a vendor control panel or a connection to their own security platform. Also, many EDR solutions include a traditional antivirus functionality and the ability for responders to remotely access compromised systems for remediation.
 
-.. include:: EDR-Vendor-Cloud-Diagram-v21.01.png
+.. image:: /_static/EDR-Vendor-Cloud-Diagram-v21.01.png
```
Comment (jungshadow) [inline docs/source/BPs/endpoint_protection.rst:16]: Images need to be loaded with the image directive.

File: docs/source/BPs/endpoint_protection.rst [hunk 2]
```diff
@@ -9,28 +9,30 @@
...
+   :alt: Vendor cloud diagram
```
Comment (jungshadow) [inline docs/source/BPs/endpoint_protection.rst:17]: The image directive also allows you to add alt text descriptions.

File: docs/source/conf.py [hunk 3]
```diff
@@ -154,8 +154,8 @@
 # documentation.
 #
 html_theme_options = {
-    'light_logo': 'cis-logo.png',
-    'dark_logo': 'cis-logo.png', # TODO: Need dark logo
+    'light_logo': 'EI-ISAC-Logo-Stack-2Spot-RGB.png',
```
Comment (jungshadow) [inline docs/source/conf.py:157]: I updated the light and dark logos while I was updating the variables.

File: docs/source/conf.py [hunk 4]
```diff
@@ -326,12 +326,14 @@
 # Global variables for use in all pages
 #
 contact_email = 'controlsinfo@cisecurity.org'
+eiisac_email = 'elections@cisecurity.org'
```
Comment (jungshadow) [inline docs/source/conf.py:329]: I added the EI-ISAC email address.

File: docs/source/index.rst [hunk 5]
```diff
@@ -35,15 +34,15 @@ CIS's Essential Guide for Election Security
...
+   BPs/public_asset_scanning
    BPs/websites
 
 .. toctree::
    :maxdepth: 1
    :caption: Appendices
 
-   Appendices/appendixindex
-   Appendices/howtoguide
+   Appendices/appendix_index
```
Comment (jungshadow) [inline docs/source/index.rst:44]: Some of these references were incorrectly named and broke the links.

File: docs/source/index.rst [hunk 6]
```diff
@@ -22,10 +22,9 @@ CIS's Essential Guide for Election Security
 
    BPs/bp_intro
    BPs/bp_index
-   BPs/albert
```
Comment (jungshadow) [inline docs/source/index.rst:25]: This page didn't seem to exist, so I removed it.

**Review by jungshadow — APPROVED** [review]
I made some edits to the file and added my reasoning in the comments. I merged this branch with main already, so it should be up-to-date.

File: docs/source/BPs/endpoint_protection.rst (hunk 2 above)
Comment (jungshadow) [reply docs/source/BPs/endpoint_protection.rst:17]: @JTAGulous You probably want to add a better image description than the one I provided 😆

**Comment by jungshadow** [comment]
Once this is merged, this PR will close #38 as well.

**Comment by JTAGulous** [comment]
I believe there are still changes within this haven't been accepted. Is this correct?

**Pull request merged**